webapp/src/routes/SpendTools/index.js

```diff
@@ -0,0 +1,358 @@
+import React, { memo } from 'react'
+import { makeStyles } from '@mui/styles'
+import Select from '@mui/material/Select'
+import Button from '@mui/material/Button'
+import { useTranslation } from 'react-i18next'
+import InputLabel from '@mui/material/InputLabel'
+import { IconButton, MenuItem, Modal, TextField, Tooltip } from '@mui/material'
```

Comment:
Try to use one level MUI import just for consistency 

in order to maintain the performance the idea is to use the top-level import

please check it: https://mui.com/material-ui/guides/minimizing-bundle-size/#option-one-use-path-imports